docs/examples.rst

```diff
@@ -115,6 +115,16 @@ services are also in the making (Sumo). The example below show how fmu-dataio ca
 to simplify an existing aggregation service, as well as make de-centralized methods more
 robust by centralizing the definitions and handling of metadata.
 
+.. note::
+    Some surfaces are exported out of RMS with NaN (UNDEF) instead of the value 0, e.g.
+    volumetrics maps. When these maps are used in post-processing across multiple realisations,
+    this can create erroneous results. Therefore, when exporting surfaces of this type,
+    set the `nan_is_zero` argument to `True` when exporting. This tells later consumers
+    of the surfaces that they should handle `NaN` as zero. Also note that the recommended
+    pattern is to make sure, prior to export, that exported surfaces have values reflecting
+    their _actual_ values. In other words: We recommend you to actively set the correct
+    values before export, but we offer this solution if you are unable to.
```

Comment:
I propose some rewording here:

It is common  that a surface exported from RMS or other sources have undefined areas. For some of surfaces, typically various thickness surfaces (e.g. HCPV thickness) from RMS volumetric jobs, the undefined values shall be treated as zero (0.0) when doing aggregations or statistics. Therefore, when exporting surfaces of this type,
set the `undef_is_zero` argument to `True` when exporting. This tells later consumers of the surfaces that they should handle `UNDEF` as zero. 

(I removed alternative preprocessing, setting `undef_as_zero` as True makes no harm even if the user prefer to do so in advance, but that id for expert users possibly)